src/main/java/com/eerussianguy/blazemap/feature/BlazeMapFeaturesClient.java

```diff
@@ -172,6 +173,7 @@ public static void initWaypoints() {
         IEventBus bus = MinecraftForge.EVENT_BUS;
         bus.addListener(EventPriority.HIGHEST, WaypointService::onServerJoined);
         bus.addListener(WaypointService::onDeath);
+        bus.addListener(WaypointSharing::onChatReceive);
```

Comment:
might be a bit nitpicky, but this should be called `onChatReceived` ... one letter missing :p